config/peerpods/podvm/lib.sh

```diff
@@ -110,9 +120,9 @@ install_binary_packages() {
             fi
 
             echo "Marking  ${install_dir}/${package_name} executable"
-            # yq extracted file name is yq_linux_amd64. Rename it
+            # yq extracted file name is yq_linux_${ARCH}. Rename it
             [[ "${package_name}" == "yq" ]] &&
-                mv "${install_dir}/yq_linux_amd64" "${install_dir}/yq"
+                mv "${install_dir}/yq_linux_${ARCH}" "${install_dir}/yq"
```

Comment:
This will fail for x86_64 since ARCH will be `x86_64` whereas the expected filename has `amd64`